client/src/components/Artists/Artist.js

```diff
@@ -30,6 +32,17 @@ export default function Artist({
   saveSong,
   savedSongs,
 }) {
+  console.log({ songUrl })
+
+  useEffect(() => {
+    if (songUrl === null) {
+      updateTrack()
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [songUrl])
+
+  if (!songUrl) return <></>
```

Comment:
Self-closing tag might be enough?

What is the difference between ``songUrl === null`` and ``!songUrl``? Isn`t ``null`` a falsy value as well? Would like to understand it :)